### PR TITLE
make it into an npm package, some cleanup, write a test that checks t…

### PR DESCRIPTION
**Pull request by moodysalem**
…he contract is deployed (it can't deploy yet)

### CODE REVIEW

File: contracts/Bounty0xCrowdsale.sol [hunk 1]
```diff
@@ -52,7 +52,7 @@ contract Bounty0xCrowdsale is Pausable, TokenController {
 
     // Vesting conditions
     uint public constant TEAM_VESTING_CLIFF = 1 weeks;                  // 1 week vesting cliff for founders and advisors
-    uint public constant TEAM_VESTING_PERIOD = 52 weeks;                // 1 year vesting period for founders and advisors 
+    uint public constant TEAM_VESTING_PERIOD = 52 weeks;                // 1 year vesting period for founders and advisors
```
Comment (dnzdlklc) [inline contracts/Bounty0xCrowdsale.sol]: Not sure why these are showing as a change. Seems the same to me.

File: truffle.js [hunk 2]
```diff
@@ -1,17 +1,14 @@
-var HDWalletProvider = require("truffle-hdwallet-provider");
-var mnemonic = "state couch illness ritual step rose west magnet arrange popular circle slam";
+require('babel-register');
+require('babel-polyfill');
+
+const HDWalletProvider = require('truffle-hdwallet-provider');
+const mnemonic = 'state couch illness ritual step rose west magnet arrange popular circle slam';
 
 module.exports = {
   networks: {
```
Comment (dnzdlklc) [inline truffle.js]: We may still require 'development' configuration to use "TestRPC" probably better to keep it as an option.

Comment (moodysalem) [reply truffle.js]: In truffle v4 if you exclude it it will launch its own testrpc

File: contracts/Bounty0xCrowdsale.sol (hunk 1 above)
Comment (moodysalem) [reply contracts/Bounty0xCrowdsale.sol]: Whitespace was removed from the end of the line by my IDE, ignore whitespace changes should help

File: truffle.js (hunk 2 above)
Comment (dnzdlklc) [reply truffle.js]: oh cool! I need to upgrade my truffle in that case.

Comment (moodysalem) [reply truffle.js]: no need as long as you do `npm install` and `npm test` to run tests

**Pull request merged**